lib/config/options/index.ts

```diff
@@ -2725,6 +2725,14 @@ const options: RenovateOptions[] = [
     globalOnly: true,
     default: [],
   },
+  {
+    name: 'platformGitCredentialsFile',
+    description:
+      "If set to `true` then the repository URL doesn't contain the authentication token, and the .git-credentials file will be used.",
```

Comment:
```suggestion
      'If set to `true` then the `~/.git-credentials` file will be used for platform authentication instead of embedding credentials in the URL.",
```